doc/article/en-US/angular1-to-aurelia.md

```diff
@@ -0,0 +1,526 @@
+</code-listing>
+
+#### Converting `ng-include` to `<compose>`
+
+<code-listing heading="Angular View Composition">
+	<source-code lang="HTML">
+		<div data-ng-include="myView"></div>
+	</source-code>
+</code-listing>
+
+<code-listing heading="Aurelia View Composition">
+	<source-code lang="HTML">
+		<compose view-model="myViewModel" view="myView"></compose>
+	</source-code>
+</code-listing>
+
+### Converting Angular directives to Custom Elements and Attributes
+
+Finally, the meat and potatoes - it's time to migrate our Angular directives.
+
+Angular and Aurelia both support the concept of extending HTML by creating custom elements and attributes. However, Aurelia handles the creation of these in a different way.
+
+#### Converting a directive to an Aurelia custom element
+
+Creating a custom element in Angular required us to write an Angular directive; we would return a configuration object that told Angular how to create our object through object properties including `scope`, `restrict`, `template`, `templateUrl`, and `link`.
+
+Instead of a configuration object, Aurelia takes a class instance.
+
+An Angular directive may have looked like this:
+
+<code-listing heading="Angular Custom Components">
+	<source-code lang="ES 2015/2016">
+		angular.directive('display', ['$window', '$document', 'animate', function ($window, $document, animate) {
+			return {
+				scope: true,
+				restrict: 'A',
+				templateUrl: 'myTemplate.html',
+				link: function (scope, element, attrs) {
+					var windowheight = angular.element($window).height();
+		
+					scope.holeHeight = function (depth) {
+						return (depth * 0.2) < windowheight ? (depth * 0.2) + 'px' : undefined;
+					};
+		
+					scope.bgPos = function (depth) {
+						return (depth * 0.2) < windowheight ? 0 : (depth - windowheight) * 0.2;
+					};
+		
+					scope.holeWidth = function () {
+						return Math.min(700, Math.max(350, _(scope.shop)
+							.reduce(function (p, c) {
+								return p + c.owned * c.digValue;
+							}, 0)
+						));
+					};
+		
+					scope.displayArray = [];
+		
+					scope.$watch('shop', function () {
+						scope.displayArray = _.values(scope.shop).reverse();
+					});
+		
+					(function tick(timestamp) {
+						angular.element($window).scrollTop($document.height());
+						animate(tick);
+					})();
+				}
+			};
+		}])
+	</source-code>
+</code-listing>
+
+Converting this directive to an Aurelia custom element requires us to remove the scope references, migrate from `link` and `controller` properties to lifecycle methods, and simplify our object. Finally, we'll have to use Aurelia's decorator functions to register our object and its configuration as a custom element instance.
+
+<code-listing heading="Aurelia Custom Components">
+	<source-code lang="ES 2015/2016">		export class Display {
+			constructor (window, document, animate) {
+				this.window = window;
+				this.document = document;
+				this.animate = animate;
+			}
+			
+			attached () {
+
+			}
+		}
```

Comment:
**Serious deficiency**. You should make the Aurelia class as similar to Angular directive. It is not fair to pick not the most trivial Angular directive as an example and show just a class constructor for Aurelia class that won't even work as custom element (not to mention functioning similarly).